geomeppy/geom/polygons.py

```diff
@@ -104,6 +105,11 @@ def bounding_box(self):
         bbox = Polygon3D([top_left, bottom_left, bottom_right, top_right])
         return invert_align_face(self, bbox)
 
+    def buffer(self, distance=None, join_style=2):
```

Comment:
I've forgotten what `join_style` means. Could we add a docstring here pls? Also mypy type hint - it should be `# type: (Optional[float]), Optional[int]) -> Polygon2D`. You'll also need `from typing import Optional`

Reply:
My bad. It should be `# type: (Optional[float], Optional[int]) -> Polygon2D`

Reply:
Oops! My bad too, I just copied and pasted what you posted, I should have noticed that there was one too many parenthesis! I just committed again. Please let me know if you need anything else revised :).